src/firefly/java/edu/caltech/ipac/firefly/server/servlets/HttpServCommands.java

```diff
@@ -33,6 +38,8 @@ public void processRequest(HttpServletRequest req, HttpServletResponse res, SrvP
 
             String fileName = sp.getOptional("file_name");
 
+            String tableSave = sp.getOptional("table_save"); //if this is true/exists, don't want to download tbl, instead load it into a tmp file
```

Comment:
Can you change `table_save` param to something else since the `cmd` is also tableSave.
Maybe `save_to_temp`?